botx/bot/bot.py

```diff
@@ -1047,6 +1052,46 @@ async def add_sticker(
 
         return botx_api_sticker.to_domain()
 
+    async def iterate_by_sticker_packs(
+        self,
+        *,
+        bot_id: UUID,
+        user_huid: UUID,
+    ) -> AsyncIterable[StickerPackFromList]:
+        """Iterate by user sticker packs.
+
+        Args:
+            bot_id: Bot which should perform the request.
+            user_huid: User huid.
+
+        Yields:
+            Sticker pack.
+        """
+
+        after = None
+
+        method = GetStickerPacksMethod(
+            bot_id,
+            self._httpx_client,
+            self._bot_accounts_storage,
+        )
+
+        while True:
+            payload = BotXAPIGetStickerPacksRequestPayload.from_domain(
```

Comment:
Also add test which ensures that next page loads correctly. You can use `monkeypatch` fixture to patch `STICKER_PACKS_PER_PAGE`.